addons/website_slides/models/slide_channel.py

```diff
@@ -54,6 +58,36 @@ def _recompute_completion(self):
             for user in users:
                 users.add_karma(partner_karma[user.partner_id.id])
 
+    def send_completed_mail(self):
+        self.ensure_one()
+        if self.channel_id.completed_template_id:
+            self.env['mail.mail'].sudo().create(self._prepare_mail_values())
+
+    def _prepare_mail_values(self):
+        subject = self.env['mail.render.mixin']._render_template(self.channel_id.completed_template_id.subject, 'slide.channel.partner', self.ids, post_process=True)[self.id]
+        body = self.env['mail.render.mixin']._render_template(self.channel_id.completed_template_id.body_html, 'slide.channel.partner', self.ids, post_process=True)[self.id]
+        mail_values = {
+            'email_from': self.channel_id.user_id.email_formatted,
+            'author_id': self.channel_id.user_id.partner_id.id,
+            'subject': subject,
+            'body_html': body,
+            'auto_delete': True,
+            'recipient_ids': [(4, self.partner_id.id)]
+        }
+        try:
+            template = self.env.ref('mail.mail_notification_light', raise_if_not_found=True)
```

Comment:
I see in hr_appraisal they don't check that "mail_notification_light" is existing before using it.
As it's a "data template", that surely breaks a lot of other stuff if deleted, it's probably safe to assume it's there.

Reply:
Nevermind they are refactoring appraisal and the new version contains a check for this template, so let's keep it after all.